contents/handbook/help.md

```diff
@@ -16,76 +24,84 @@ This will require:
 
 Everything else is a means to this end! We often do onboarding in person to accelerate all the above. This usually takes around a week.
 
-## General etiquette
+## Ask for help, but only after you've tried first
 
-### Don't assign issues to people
+If you've just joined us, there's a lot you probably don't know. That's okay! However, we _do_ expect that you try to help yourself. Here's a framework to use as a guide:
 
-You can list and categorize issues.
+- If it's one of our own products that we build and sell:
+  - We expect you to [read our docs](/docs).
+  - We expect you to search through GitHub or Slack.
+  - If it's in beta, it might be buggy or missing features. Definitely request/report them, but also figure out how to do your job with plan B and plan C if it won't be fixed quickly - this is what we generally call grit.
```

Comment:
```suggestion
  - If it's in beta, it might be buggy or missing features. Definitely request/report them, but also figure out how to do your job with plan B and plan C if it won't be fixed quickly - this is what we generally call grit.
  - For engineers: try to figure something out by yourself for at least 1 hour, but don't remain totally stuck on something for more than 2 hours before asking for help. This time window should increase over time as you run into more questions that likely no one has the answer to, at which point it's time to dig in and figure it out.
```